<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NDT Static Bending Strength of Wood</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>1491</width>
      <height>701</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_bg">
     <attribute name="title">
      <string>SINYAL BACKGROUND</string>
     </attribute>
     <widget class="QPushButton" name="pbBG">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">b</string>
      </property>
      <property name="text">
       <string>Rekam</string>
      </property>
     </widget>
     <widget class="QComboBox" name="pilihMD">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="pilihMJ">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>20</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QGraphicsView" name="mdBG">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>651</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="mjBG">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>50</y>
        <width>651</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="fftBG">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>1431</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelMD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pilih MD</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMJ">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pilih MJ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_kayu">
     <attribute name="title">
      <string>SINYAL KAYU</string>
     </attribute>
     <widget class="QPushButton" name="pbKayu">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>40</y>
        <width>93</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rekam</string>
      </property>
     </widget>
     <widget class="QComboBox" name="pilihMDK">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="pilihMJK">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>40</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QGraphicsView" name="mdKayu">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>621</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="mjKayu">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>70</y>
        <width>621</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="fftKayu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>1471</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="namaKayu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="labelKayu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nama Kayu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMDK">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pilih MD</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMJK">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pilih MJ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>211</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:1px solid rgb(184, 184, 184);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Frekuensi:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="frequency_label">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Kelas:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="kelas_label">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Kayu:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pbKayu_analysis">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>93</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Analysis</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_af">
     <attribute name="title">
      <string>ANALISIS FREKUENSI</string>
     </attribute>
     <widget class="QGraphicsView" name="fftMulti">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>1231</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbUpload">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Unggah File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbHapus">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
     <widget class="QListWidget" name="file_list">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>771</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbRefresh">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="judul">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SISTEM UJI TIDAK MERUSAK KETEGUHAN LENTUR STATIK KAYU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1515</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
